src/ContentHubClient.php

```diff
@@ -940,12 +956,15 @@ public function updateWebhook($uuid, array $options) {
    *
    * @param string $webhook_uuid
    *   Webhook uuid.
+   * @param string $suppress_duration
+   *   Duration for which webhook should be suppressed. e.g. 24h.
```

Comment:
Perhaps it is worth a comment if we leave the suppressed_duration blank, it will indefinitely suppress the webhook.